src/dyn_sensitivity.jl

```diff
@@ -176,11 +193,20 @@ end
 Compute the part of the Jacobian associated with charge and discharge, 
 with `dims` being the dimension of the static system and `n` the number of nodes.
 """
-function compute_jacobian_kkt_charge_discharge(dims, n)
+function compute_jacobian_kkt_charge_discharge_ramp(dims, n, l)
```

Comment:
Update doc

Reply:
And what is the rationale here for combining both charge_discharge and ramp in the jacobian? You could have built them separately right? (not saying it would be better, just trying to understand)

Reply:
Only because the ramping constraints are added with the storage constraints, so it makes sense to group them together elsewhere.